tests/integration/test_github_app.py

```diff
@@ -79,3 +84,14 @@ def test_get_project_having_key_as_path(self):
         finally:
             if not self.github_app_private_key_path:
                 Path(github_app_private_key_path).unlink()
+
+    def test_github_proj_no_app_creds(self):
+        service = GithubService(
+            github_app_id="123", github_app_private_key=TESTING_PRIVATE_KEY
+        )
+        project = GithubProject("packit", service, "packit-service")
```

Comment:
I wouůd use keyword arguments to be sure what's what...

Reply:
You would, but I won't - this is perfectly clear to me:

https://github.com/packit-service/ogr/blob/5bc1dcf352e37c621d9a6a58225325fec85d0c21/ogr/services/github/project.py#L64

there is also the assert which makes sure the namespace and repo name are correct.

Reply:
You need to check `__init__` to get that info. And the order of attributes is misleading.

I know what's what, but this is much clearer for anyone:

```suggestion
        project = GithubProject(service=service, namespace="packit-service", repo="packit")
```

Reply:
The ordering of these arguments is a little bit weird: "service, namespace, repo" or "namespace, repo, service" would be much easier to guess. But given the current status +1 for being explicit with this in order to improve readability.